src/engine/strat_engine/engine.rs

```diff
@@ -122,6 +143,41 @@ impl Engine for StratEngine {
     fn pools(&self) -> Vec<&Pool> {
         self.pools.into_iter().map(|x| x as &Pool).collect()
     }
+
+    fn get_eventable(&mut self) -> EngineResult<Option<Box<Eventable>>> {
+        Ok(Some(Box::new(DM::new()?)))
+    }
+
+    fn evented(&mut self) -> EngineResult<()> {
+        let device_list = DM::new()?.list_devices()?;
+
+        for &(ref dm_name, _, new_event_nr) in &device_list {
```

Comment:
Either we'll be using a newer version of devicemapper, i which case every single one of these ```new_event_nr``` values will be ```Some``` or we'll not, in which case they will all be ```None```. In the second case, no pool will ever be "evented on".

We should definitely document what our stance is on that situation.

Reply:
Yeah this is a problem. Or at least a weirdness. Maybe if `None` we should instead *always* call event_on() for all watched devs?

Reply:
I think that if devicemapper version is early so that no event nrs available, then the only meaningful thing is to call ```event_on()``` for all pools.

I think that in that same case, there will be no watched devs.